src/trace/patch-http.ts

```diff
@@ -125,13 +125,13 @@ function isIntegrationTest() {
  * HTTP GET https://ip-ranges.datadoghq.com/ Headers: ["x-datadog-parent-id:abc"] Data: {}
  * @param options The options for the HTTP request
  */
-function _logHttpRequest(options: http.RequestOptions) {
-  let headerMessage = "Headers: []";
+function _logHttpRequest(options: http.RequestOptions, traceHeaders: Partial<TraceHeaders>) {
+  let headerMessage = "TraceHeaders: []";
 
-  if (options.headers) {
-    const headerStrings = Object.entries(options.headers).map(([name, value]) => `${name}:${value}`);
+  if (traceHeaders) {
```

Comment:
Instead of logging all the headers here, only log the relevant ones ,i.e. the Trace context headers.